src/mmocore/Logger.ts

```diff
@@ -11,7 +11,7 @@ export enum LogLevel {
 export default class Logger implements ILogger {
   private _context = "";
 
-  private _logLevel: LogLevel = 1;
+  private _logLevel: LogLevel = 8;
```

Comment:
The default logger level should not be set to DEBUG

Reply:
Sorry, this is my testing leftover.